packages/create-pantheon-decoupled-kit/src/actions/convertCSSModules.ts

```diff
@@ -0,0 +1,71 @@
+import chalk from 'chalk';
+import { execSync } from 'child_process';
+import { Action, isString } from '../types';
+import autoprefixer from 'autoprefixer';
+import postcss from 'postcss';
+import fs from 'fs-extra';
+import klaw from 'klaw';
+import globalData from '@csstools/postcss-global-data';
+import customProperties from 'postcss-custom-properties';
+import { rootDir } from '..';
+
+export const getAllFiles = async (
+	templateDir: string,
+	outDir: string,
+): Promise<string[]> => {
+	const templates: string[] = [];
+
+	// loop through each dir
+	for await (const file of klaw(templateDir)) {
+		if (file.stats.isDirectory()) continue;
+		const templateName = file.path.split(outDir)[1];
+
+		templates.push(templateName);
+	}
+	return templates;
+};
```

Comment:
See comment on line 44 as I'm not sure this extra helper is needed. If it is needed:
I don't mind this logic being broken out into a helper function, but it can be written in a more generic way.
1. if it's truly a generic getAllFiles helper, why does it need two args? It should probably take 1 arg (a path to a directory) and return all files in that dir. 
2. In this case, we should be gathering already rendered _files_ not _templates_. So there should be no need to split the template name from the file path like we do in `dedupeTemplates`.
3. I would also move this to `utils` if it's meant to be a general helper utility, otherwise you can ditch the `export` and just use it locally in this file.